### ISSUE TITLE
Resource pack loading issue

### ISSUE THREAD

**Issue by Adizbek** [pmmp/PocketMine-MP#1233]
Something wrong with loading resource packs.

![photo5449861410822662194](https://user-images.githubusercontent.com/10253982/28497441-92a89e82-6fa0-11e7-8596-302fc47cdcd4.jpg)

### OS and versions
Сервер использует PocketMine-MP 1.6.2dev-267「 Unleashed」, версия API 3.0.0-ALPHA6 для Minecraft: PE v1.1.0.55 (версия протокола 113) 
* PHP: 7.0.3
* Server OS: Debian 8
* Game version: PE (Android)

**Comment by vzlomka76** [comment]
In English: 16532696,0 GB

**Comment by dktapps** [comment]
Your version is over 40 commits out of date. Please try a newer version.

If the issue is still reproducible, the following information is required:
- A list of resource packs in use on the server
- Version of the client you are using (1.1.4 has bugs with resource packs)

This issue will be closed until the above conditions have been met.

**Comment by Adizbek** [comment]
Minecraft PE client version: 1.1.3.1

Server info when I type '-version':
![image](https://user-images.githubusercontent.com/10253982/28498140-23a2841a-6fb1-11e7-8965-d6cae866f205.png)

Still issue the same? Did anyone use resources packs ?

**Comment by dktapps** [comment]
I asked for a list of resource packs that you are using.

**Comment by Adizbek** [comment]
I have downloaded my resources here: http://mcpedl.com/faithful-pe/
Link to download file: http://www.mediafire.com/file/kyb6bb5ybttfad1/Faithful+PE+1.1.3-rv1.zip

**Comment by dktapps** [comment]
I'm using that exact resource pack and have no such issue...

**Comment by Adizbek** [comment]
Maybe something wrong with php ? Which version php do you use? Does php need any lib for serving resource packs ? 

_Sent from my Lge LG-H502 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by dktapps** [comment]
PHP 7.0.21. You can update your PHP build from our [Jenkins server](https://jenkins.pmmp.io/job/PHP-7.0-Linux-x86_64/)

**Comment by Adizbek** [comment]
Still the same. Which version pe do you use.?

_Sent from my Lge LG-H502 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by Adizbek** [comment]
Maybe php needs 3rd party library to handle zip files(resource pack) ? 

_Sent from my Lge LG-H502 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by SleepSpace9** [comment]
Today I encountered the same problem. I also tested the "Faithful" resource pack and got this abnormal value for it's download size:

![grafik](https://user-images.githubusercontent.com/15960628/29641314-4a6f026a-8863-11e7-9b1e-bee099c87003.png)

I used the Phar from Jenkins (PocketMine-MP 1.6.2dev-**467** without any plugins) when this occured. Even the PHP 7.0.17 build from Jenkins didn't help.
Later I started pmmp from current source (55e32424a09fa699379c53bc0c573ec93c5b8e2f) and 🎉 the download size was displayed correctly and all worked fine.
Seems to be a platform/compiling issue (?).

_OS: Ubuntu 14.04 (64bit) / PHP 7.0.22_
